internal/vm/setup.go

```diff
@@ -61,6 +61,15 @@ _G["process_auto_fields"] = function(plugin)
   return json.encode(p)
 end
 
+_G["validate_plugin_schema"] = function(plugin_schema_string)
+  local plugin_schema = loadstring(plugin_schema_string)()
+  local ok, err_t = MetaSchema.MetaSubSchema:validate(plugin_schema)
+  if not ok then
+    return nil, tostring(errors:schema_violation(err_t))
+  end
+  return plugin_schema.name, nil
+end
```

Comment:
In future versions of `goks` we might want to consider removing the code duplication in `load_plugin_schema` and update the signature to require the `plugin_name` and `plugin_schema_string`. If we decide to keep the schema validation in the `load_plugin_schema` function we should look at using local functions and exposing them in the return in order to remove some duplicated code.